src/content/docs/it/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,423 @@
+
+Astro v2.0 non modifica più il valore di `site`. `Astro.site` utilizzerà il valore esatto che è stato definito e, se lo si desidera, la barra finale deve essere specificata.
+
+#### Cosa devo fare?
+
+ In `astro.config.mjs`, aggiungi una barra finale all'URL impostato in `site`.
+
+```js del={5} ins={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  site: 'https://example.com',
+  site: 'https://example.com/',
+});
+```
+
+### Cambiato:  Cartella `_astro/` per asset della build
+
+Nella v1.x, gli asset venivano creati in varie posizioni, tra cui `assets/`, `chunks/` e nella radice dell'output della build.
+
+Astro v2.0 sposta e unisce la posizione di tutti gli assets in una nuova cartella `_astro/`.
+
+<FileTree>
+- dist/
+  - _astro
+    - client.9218e799.js
+    - index.df3f880e0.css
+</FileTree>
+
+Puoi controllare questa posizione con la [nuova opzione di configurazione `build.assets`](/it/reference/configuration-reference/#buildassets).
+
+#### Cosa devo fare?
+
+Aggiorna la configurazione della tua piattaforma di distribuzione se quest'ultima si basa sulla posizione degli assets.
+
+### Cambiato: Configurazione dei plugin di markdown
+
+#### Rimosso: `extendDefaultPlugins`
+
+Nella v1.x, Astro usava `markdown.extendDefaultPlugins` per riattivare i plugin predefiniti di Astro quando venivano aggiunti dei propri plugin di Markdown.
+
+Astro v2.0 rimuove questa opzione completamente perchè questo comportamento ora è predefinito.
+
+L'applicazione di plug-in remark e rehype nella configurazione di Markdown **non disabilita più i plug-in predefiniti di Astro**. GitHub-Flavored Markdown e Smartypants ora vengono applicati indipendentemente dalla configurazione di `remarkPlugins` o `rehypePlugins` personalizzati.
+
+##### Cosa devo fare?
+
+Rimuovi `extendDefaultPlugins` nella tua configurazione. Ora questo è il comportamento predefinito di Astro nella v2.0, e puoi eliminare questa riga senza nessuna sostituzione.
+
+```js del={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  markdown: {
+    extendDefaultPlugins,
+  }
+});
+```
+
+#### Aggiunto: `gfm` e `smartypants`
+
+Nella v1.x, potevi scegliere se disabilitare entrambi i plugin di Markdown di Astro predefiniti (GitHub-Flavored Markdown e SmartyPants) impostando `markdown.extendDefaultPlugins: false`.
+
+Astro v2.0 sostituisce `markdown.extendDefaultPlugins: false` con delle opzioni booleane separate per controllare indipendentemente ognuno dei plugin Markdown di Astro. Questi sono abilitati di default e possono essere impostati su `false` in modo indipendente.
+
+##### Cosa devo fare?
+
+Rimuovi `extendDefaultPlugins: false` e aggiungi dei flag per disabilitare ciascun plugin indipendentemente.
+
+- `markdown.gfm: false` disables GitHub-Flavored Markdown
```

Comment:
**issue:** these 2 lines are still in English.
I would suggest:
Line 160: `disabilita GitHub-Flavored Markdown`
Line 161: `disabilita SmartyPants`